plugins/permission-react/src/apis/IdentityPermissionApi.ts

```diff
@@ -53,4 +53,11 @@ export class IdentityPermissionApi implements PermissionApi {
     );
     return response[0];
   }
+
+  async authorizeAll(requests: AuthorizePermissionRequest[]) {
```

Comment:
What do you think about adding some lightweight same-tick caching to the `authorize` method instead of adding this? Feeling that that could bring bigger benefits on top of being able to batch requests from the same source, as that'd batch requests from separate sources too.